<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="bg" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>450</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#bg{
background-color:qlineargradient(spread:pad, x1:0, y1:0.551, x2:1, y2:0.511, stop:0 rgba(0, 0, 127, 255), stop:1 rgba(85, 170, 255, 255))}
</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>450</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Report List</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="backb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius:20px;
font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 127);
}

QPushButton:hover{
	
	background-color: rgb(255, 255, 0);
}</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="reportTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>430</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
     </property>
     <column>
      <property name="text">
       <string>Report ID</string>
      </property>
      <property name="background">
       <color alpha="150">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Area</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Reason</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>-180</x>
       <y>0</y>
       <width>1081</width>
       <height>761</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../Downloads/07PlusLebuhraya_1528287033.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_5</zorder>
    <zorder>label</zorder>
    <zorder>backb</zorder>
    <zorder>reportTable</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
